apps/condo/migrations/20210831103000-0050_manual_set_can_manage_divisions_to_administrator_organization_employee_roles.js

```diff
@@ -0,0 +1,14 @@
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+        UPDATE "OrganizationEmployeeRole"
+        SET "canManageDivisions" = true
+        WHERE name = 'employee.role.Administrator.name';
```

Comment:
Maybe, we can add a field to OrganizationEmployeeRole with role type?

Reply:
I also was thinking about it and told on a standup.
But, It's redundant, because our database have clear keys.
Below is a query for names:


```postgresql
condo=# select name from "OrganizationEmployeeRole" group by name;
               name               
----------------------------------
 тест роль
 employee.role.Administrator.name
 employee.role.Manager.name
 employee.role.Dispatcher.name
 employee.role.Foreman.name
 employee.role.Technician.name
(6 rows)
```